src/assets/scss/_footer.scss

```diff
@@ -0,0 +1,216 @@
+footer {
+  background: url(../images/christmas_snow_trees-wide.png) no-repeat;
+  position: relative;
+  background-position: bottom center;
+  z-index: 0;
+}
+.contacts {
+  &::before {
+    background: url(../images/footer.png) no-repeat;
+    content: "";
+    position: absolute;
+    background-size: cover;
+    width: 100%;
+    height: 100%;
+    z-index: -1;
+  }
+
+  &__info {
+    padding: 112px 0 122px;
+    display: flex;
+    justify-content: space-between;
+    color: #c7cccf;
+    align-items: stretch;
+  }
+}
+
+.form {
+  padding: 0px 33px 35px 30px;
+  background-color: #021623b3;
+  z-index: 2;
+  margin-right: 26px;
+  width: 613px;
+
+  &__input,
+  &__subject,
+  &__email,
+  &__message {
+    display: block;
+    font-family: "Raleway";
+    color: #ffffffb3;
+    background-color: #02162300;
+    font-size: 15px;
+    outline: none;
+    border: none;
+    border-bottom: solid 1px #ffffff4d;
+    line-height: 3.2;
+    width: 100%;
+  }
+
+  &__input {
+    margin: 27px 0 0px;
+    padding: 0 4px;
+  }
+  &__subject {
+    margin: 11px 0 0px;
+    padding: 0 4px;
+  }
+  &__email {
+    margin: 10px 0 0px;
+    padding: 0 4px;
+  }
+  &__message {
+    margin: 13px 0 0px;
+    padding: 0 4px 26px;
+  }
+  &__submit {
+    display: inline-block;
+    padding: 14px 35px 13px 35px;
+    font-size: 18px;
+    color: rgba(0, 0, 0, 0.7);
+    outline: none;
+    border: none;
+    background-color: #6eeee4;
+    border-radius: 4px;
+    margin: 36px 0 2px;
+    line-height: 1.556;
+    letter-spacing: 2.6px;
+  }
+
+  &__submit:hover {
+    opacity: 0.7;
+    transition: 0.4s;
+  }
+}
+
+.form__input::-webkit-input-placeholder,
+.form__subject::-webkit-input-placeholder,
+.form__email::-webkit-input-placeholder,
+.form__message::-webkit-input-placeholder {
+  color: #ffffff9a;
```

Comment:
You mustn't add the vendor prefix manually

Reply:
Thanks, I fixed if.
``` suggestion
  &__input::placeholder,
  &__subject::placeholder,
  &__email::placeholder,
  &__message::placeholder {
    color: #ffffff9a;
  }
```